mmcv/cnn/rfsearch/operator.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+import logging
+from typing import Dict
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+from .utils import expands_rate, value_crop
+
+logger = logging.getLogger('Operators')
+logger.setLevel(logging.INFO)
```

Comment:
mmcv provides `get_logger` in `mmcv.utils.logging`, which generates logger with regular format.